packages/devtools_app/test/memory/profile/allocation_profile_table_view_test.dart

```diff
@@ -90,6 +95,10 @@ void main() {
         expect(find.text('Total'), findsOneWidget);
         expect(find.text('Old Space'), findsOneWidget);
         expect(find.text('New Space'), findsOneWidget);
+        expect(find.text('Usage'), findsNWidgets(3));
+        expect(find.text('Capacity'), findsNWidgets(3));
+        expect(find.text('Collections'), findsNWidgets(3));
+        expect(find.text('Latency'), findsNWidgets(3));
```

Comment:
should you check that the values for these columns from your sample data are here as well?